infinispan-impl/src/main/java/org/infinispan/arquillian/core/RESTEndpoint.java

```diff
@@ -57,7 +57,7 @@ public InetAddress getInetAddress()
       String hostname;
       try
       {
-         hostname = MBeanUtils.getMBeanAttribute(provider, mBeans.getHorRodServerMBean(), ServerModuleAttributes.HOST_NAME);
+         hostname = MBeanUtils.getMBeanAttribute(provider, mBeans.getRestServerMBean(), ServerModuleAttributes.HOST_NAME);
          return InetAddress.getByName(hostname);
       }
       catch (Exception e)
```

Comment:
There's an exception saying "Could not retrieve HotRod host". It should be changed to "REST"